pkg/model/crd.go

```diff
@@ -303,6 +303,20 @@ func (r *CRD) IsSecretField(path string) bool {
 	return false
 }
 
+// GetImmutableFields returns list of immutable fields present in CRD
```

Comment:
Technically, what this is returning is a list of **field paths** that are marked immutable, not fields...